app/controllers/user/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
Please go through all the controllers devise has added and list down in a doc the purpose of each action in each of these controllers
Ex:
sessions#new -> user to render the sign in page
....

Reply:
ok sir

Reply:
please share the doc that you have prepared